SwSpotify/spotify.py

```diff
@@ -1,5 +1,7 @@
 import sys
+import os
 from SwSpotify import SpotifyClosed, SpotifyPaused
+from . import spotify_web
```

Comment:
from SwSpotify import spotify_web is preferable so just tack it onto the previous line